new/analyse/spec/add-dependencies.spec.ts

```diff
@@ -93,4 +88,122 @@ describe('addDependencies Module', () => {
       expect(result).toEqual(expectedEnv);
     });
   });
+
+  describe('with rlsr.json data', () => {
```

Comment:
can you add a `without rlsr.json data` section as well?

Reply:
That's already covered by your tests above, isn't it?

Reply:
I think they are. I think it'll be better to have them wrapped in a `describe('without rlsr.json data', () => { ... })`

Reply:
Cool. Changed that line.